packages/core/src/js/feedback/FeedbackWidget.types.ts

```diff
@@ -247,6 +262,23 @@ export interface FeedbackWidgetStyles {
   sentryLogo?: ImageStyle;
 }
 
+/**
+ * The props for the feedback button
+ */
+export interface FeedbackButtonProps extends FeedbackButtonTextConfiguration {
+  showFeedbackButton?: boolean;
```

Comment:
I would replace the `showFeedbackButton` option with static API `Sentry.showFeedbackButton()` and `Sentry.hideFeedbackButton()` to allow for more dynamic scenarios, like showing the button when an error happens or when shake gesture is detected.

The styles, I think, make sense to set in the integration option as it is.

The hide, I think, is important in this case to allow for hiding the button after the feedback is submitted, for example.

Reply:
Good idea Krystof 👍 
Updated with 06adadd8557e2788e70cc9dcb4071aa3252e3823